modules/serverless-pingsource-kn.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-pingsource.adoc
+
+[id="serverless-pingsource-kn_{context}"]
+= Using PingSource with the `kn` CLI
+
+The following sections describe how to create, verify and remove a basic PingSource using the `kn` CLI.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log:
```

Comment:
Will the user know what _its_ means? The service? PingSource?

Reply:
Yes, PingSource is described in the assembly, and Knative services have their own sections in the docs (it's basically a user created application) :slightly_smiling_face: I think we should be OK here.